### PR TITLE
#18e5hp menu

### PR DESCRIPTION
**Pull request by ZTKpro**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,12 +1,24 @@
 <template>
   <div id="app">
-    <h1>BridgeIT</h1>
+    <Navi/>
```
Comment (hayuna) [inline src/App.vue]: Are you sure that `Navi` is proper name for tag?

Navi means US Marines

File: src/App.vue [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
   </div>
 </template>
 <script>
-export default {};
+import Navi from '@/components/nav.vue';
+export default {
+  name: "App",
```
Comment (hayuna) [inline src/App.vue]: Do you use this props in this project? Could you explain that?

File: src/App.vue [hunk 3]
```diff
@@ -1,12 +1,24 @@
...
+  components: {
+    Navi
+  }
+}
 </script>
 
 <style lang="scss">
 @import "@/assets/style/modules/button.scss";
+body {
```
Comment (hayuna) [inline src/App.vue:16]: I think these styles were using in globals.scss. Isn't that redundant?

File: src/assets/style/modules/button.scss [hunk 4]
```diff
@@ -9,18 +9,22 @@
...
     background: $primary;
+
     &:hover {
       background: $primaryh;
       border: 2px solid $primaryh;
     }
+
     &:active {
       background: $primarya;
       border: 2px solid $primarya;
```
Comment (hayuna) [inline src/assets/style/modules/button.scss:23]: `primarya` and `primaryh` are confusing for me? What about rest of viewers? Are you understand these names?

File: src/assets/style/variables/colors.css [hunk 5]
```diff
@@ -0,0 +1 @@
+null
```
Comment (hayuna) [inline src/assets/style/variables/colors.css]: null? Could you explain that?

File: src/assets/style/variables/colors.min.css [hunk 6]
```diff
@@ -0,0 +1 @@
+null
```
Comment (hayuna) [inline src/assets/style/variables/colors.min.css]: null? Could you explain that?

File: src/assets/style/variables/colors.scss [hunk 7]
```diff
@@ -20,3 +20,6 @@ $secondarya: #0420af;
 $yellow: #f2db07;
 $green: #07f255;
 $red: #f20732;
+a{
```
Comment (hayuna) [inline src/assets/style/variables/colors.scss]: Why do you use empty tag?

File: src/components/nav.vue [hunk 8]
```diff
@@ -0,0 +1,199 @@
...
+ <div class="nav_wrapper">
+
+    <div class="nav_changer">
+     <div><p>Home</p></div>
+     <div  class="active"><p>Workspace</p></div>
+     <div><p>Others</p></div>
+    </div>
+
+    <div class="nav_iconbox">
+      <img src="http://serwer1982875.home.pl/img-bright/search_iconn.svg" alt="search icon">
```
Comment (hayuna) [inline src/components/nav.vue:11]: Are you sure that using temporary server is a good idea? Better will be use assets or some ready components with sag icon

**Review by hayuna — CHANGES_REQUESTED** [review]
@AzotoN1 @hevowy Could you check this pull request?

File: src/App.vue (hunk 1 above)
Comment (ZTKpro) [reply src/App.vue]: When i try nav be a conflict with html5 tag

File: src/App.vue (hunk 3 above)
Comment (ZTKpro) [reply src/App.vue:16]: i forget delete this.

File: src/components/nav.vue (hunk 8 above)
Comment (ZTKpro) [reply src/components/nav.vue:11]: I use the best option for me.  When we have serve  change this.

File: src/assets/style/variables/colors.scss (hunk 7 above)
Comment (ZTKpro) [reply src/assets/style/variables/colors.scss]: My bad. I forget delete this.

File: src/App.vue (hunk 1 above)
Comment (hayuna) [reply src/App.vue]: What about Navigation or something?

File: src/components/nav.vue (hunk 8 above)
Comment (hayuna) [reply src/components/nav.vue:11]: You can download these files (if license allows you of course) and store it in assets directory

File: src/assets/style/modules/button.scss (hunk 4 above)
Comment (dam-orzel) [reply src/assets/style/modules/button.scss:23]: This is my naming of these properties. h-hover a-active. We have the last moment to standardize the names. Please create a standardization for the naming of global variables on clickup

File: src/components/nav.vue [hunk 9]
```diff
@@ -0,0 +1,217 @@
+<template>
+ <div class="nav_wrapper">
```
Comment (dam-orzel) [inline src/components/nav.vue:2]: Why are there no corresponding semantic html tags in the entire project? It lacks even as basic as nav

**Review by dam-orzel — CHANGES_REQUESTED** [review]

**Pull request closed without merging**